core/py/src/braintrust_core/span_identifier_v1.py

```diff
@@ -0,0 +1,139 @@
+# Serialization format for capturing all relevant information about a span
```

Comment:
Just a copy of the old `span_identifier.py` file with everything renamed to suffix with `V1`